### PR TITLE
Make damageable mobs burn to ashes on too much heat damage

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

1500 heat damage and mob will turn in ash

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Mobs will burns to ashes if they reach 1500 heat damage.

### CODE REVIEW

**Comment by Hanzdegloker** [comment]
no more 5.6k heat damage CE

Assuming this applies to players yes?

**Comment by Dutch-VanDerLinde** [comment]
i think it would be cooler if it turned you into a charred corpse rather than just ash

File: Resources/Prototypes/Entities/Mobs/base.yml [hunk 1]
```diff
@@ -60,6 +60,16 @@
         damage: 400
       behaviors:
       - !type:GibBehavior { }
+    - trigger:
+        !type:DamageTypeTrigger
+        damageType: Heat
+        damage: 1500
+      behaviors:
+      - !type:SpawnEntitiesBehavior
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Mobs/base.yml:68]: wont this not delete the mob

**Comment by UbaserB** [comment]
What about humans, will they turn to husks?

**Comment by QuietlyWhisper** [comment]
This didn't work when I tried to test it

File: Resources/Prototypes/Entities/Mobs/base.yml (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Prototypes/Entities/Mobs/base.yml:68]: All this currently does is spawn an ash entity when the mob reaches the threshhold

Comment (lzk228) [reply Resources/Prototypes/Entities/Mobs/base.yml:68]: ah my bad

**Comment by QuietlyWhisper** [comment]
Working on a burn body behavior to support this

**Pull request closed without merging**